src/models/simulation.ts

```diff
@@ -58,22 +58,22 @@ const defaultPressureSystems: IPressureSystemOptions[] = [
   {
     type: "high",
     center: {lat: 34, lng: -29},
-    strength: 1500000
+    strength: 15
```

Comment:
I may have missed it but is there a note on what these numbers represent or how they're scaled?

Reply:
Thanks. I mentioned that in a few places, but I've just added a comment to IPressureSystemOptions, as it seems the best place to document it:
https://github.com/concord-consortium/hurricane-model/pull/12/commits/c6a453fc374c12241917649e8982b3cffc7c6b67